File: testprojects/master_scala_mainone/pipelines/SC_CONFIG_PIP/code/src/main/scala/io/prophecy/pipelines/sc_config_pip/graph/Reformat_2.scala
package io.prophecy.pipelines.sc_config_pip.graph

import io.prophecy.libs._
import io.prophecy.pipelines.sc_config_pip.config.ConfigStore._
import io.prophecy.pipelines.sc_config_pip.config.Context
import io.prophecy.pipelines.sc_config_pip.udfs.UDFs._
import io.prophecy.pipelines.sc_config_pip.udfs._
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object Reformat_2 {

  def apply(context: Context, in: DataFrame): DataFrame =
    in.select(
      concat(lit(context.config.c_string), col("`c  - int`")).as("c_config")
    )

}

File: testprojects/python_main/pipelines/PYTHON_ACL_HC_SHARED/code/com/sg_src/main/graph/Reformat_2.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from prophecy.libs import typed_lit
from com.sg_src.main.config.ConfigStore import *
from com.sg_src.main.udfs.UDFs import *

def Reformat_2(spark: SparkSession, in0: DataFrame) -> DataFrame:
    return in0.select(col("a"), concat(lit(Config.c_string), col("a")).alias("config_use"))
